app/src/main/java/com/codingblocks/cbonlineapp/mycourse/overview/OverviewFragment.kt

```diff
@@ -49,9 +54,11 @@ class OverviewFragment : BaseCBFragment(), AnkoLogger {
 
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
+        courseExtensionsRv.setRv(requireContext(), extensionListAdapter)
         viewModel.run?.distinctUntilChanged()?.observer(viewLifecycleOwner) { courseAndRun ->
             viewModel.runStartEnd = Pair(courseAndRun.runAttempt.end.toLong() * 1000, courseAndRun.run.crStart.toLong())
             viewModel.runId = (courseAndRun.run.crUid)
+            viewModel.fetchExtensions(courseAndRun.course.cid)
```

Comment:
@aggarwalpulkit596 Is this the `productId` for a course?

Reply:
no it will be the runId

Reply:
Also, this seems to give the desired results for `course.productId`